minicluster/src/test/resources/log4j2-test.properties

```diff
@@ -33,6 +33,6 @@ logger.01.level = error
 logger.02.name = org.apache.zookeeper
 logger.02.level = error
 
-rootLogger.level = info
+rootLogger.level = debug
```

Comment:
Is this temporary? I hope this doesn't make our tests during the build too spammy.